src/css/home.scss

```diff
@@ -1,6 +1,38 @@
-.home-search {
-  padding: 0;
-  width: 445px;
+$spacer1: map-get($spacers, 1);
+$spacer2: map-get($spacers, 2);
+$spacer3: map-get($spacers, 3);
+$spacer4: map-get($spacers, 4);
+$spacer5: map-get($spacers, 5);
+$spacer6: map-get($spacers, 6);
+$spacer7: map-get($spacers, 7);
+$spacer8: map-get($spacers, 8);
+
+.search-header {
+  .main-header {
+    @include media-breakpoint-down(md) {
+      padding: $spacer4 0;
+    }
+    @include media-breakpoint-up(lg) {
+      padding: $spacer7 0;
+    }
+  }
```

Comment:
was the idea of using `$spacers` here, instead of bootstrap classes like `py-lg-7`, to make sure that the padding lined up with these breakpoints?

Reply:
Hmm yea, I found it that once I started doing breakpoints to apply padding and margin, the class property starts turning into that alphabet soup that you mentioned a while back.  I thought that this might be simpler to read, although we should standardize one way or another.  As far as I know, with the class tags, there's no way to do a range of breakpoints, is there?  `media-breakpoint-up(lg)` includes both `lg` and `xl`, and `media-breakpoint-down(md) includes `md`, `sm` and `xs`, but as far as I know you can't do something like `px-6-xs-md`?

Reply:
yeah, I don't know of a good solution to that really. I think it's sort of a yucky aspect of using bootstrap for things, you end up with those super long class names. especially when we're setting `display: flex` and other properties via class names too...

I don't think that the bootstrap classes have that level of granularity unfortunately. Well, hopefully we'll find a better solution down the road!